File: tableView/tableView/ViewController.m
//
//  ViewController.m
//  tableView
//
//  Created by Nanthakumar R on 09/11/15.
//  Copyright (c) 2015 sourcebits. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
    @property (strong, nonatomic) IBOutlet UITableView *tabView;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

-(NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
    return 2;
}
@end

File: How Many Fingers/How Many Fingers/ViewController.swift
//
//  ViewController.swift
//  How Many Fingers
//
//  Created by Nanthakumar R on 13/10/15.
//  Copyright (c) 2015 sourcebits. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet var fingersNumber: UITextField!

    @IBOutlet var resultsForFingerGuessing: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    @IBAction func calculateAge(sender: AnyObject) {
        var randomNumber = arc4random_uniform(10)
        if fingersNumber.text.toInt() == Int(randomNumber) {
            
        resultsForFingerGuessing.text = "Yes, you have guessed the fingers correctly as \(fingersNumber.text.toInt())"
            
            
            
        } else {
            resultsForFingerGuessing.text = "Sorry, the correct fingures are \(Int(randomNumber)) not \(fingersNumber.text.toInt()!)"
        }
    }
    
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

